app/hooks/useQuestionStateInUrl.ts

```diff
@@ -91,9 +90,9 @@ export default function useQuestionStateInUrl(minLogo: boolean, initialQuestions
       text: null,
       answerEditLink: null,
       relatedQuestions: [],
-      ...questionMap.get(pageid),
       questionState,
       tags: [],
+      ...questionMap.get(pageid),
```

Comment:
this was overwriting the question's tags, resulting in them sometimes being shown, sometimes not